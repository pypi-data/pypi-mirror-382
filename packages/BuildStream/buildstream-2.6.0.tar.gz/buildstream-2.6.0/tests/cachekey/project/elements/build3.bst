# This tests that staging into an alternative
# prefix affects the cache key
#
kind: manual
sources:
- kind: local
  path: files/local

depends:
- elements/build1.bst
- filename: elements/build1.bst
  config:
    location: /opt
