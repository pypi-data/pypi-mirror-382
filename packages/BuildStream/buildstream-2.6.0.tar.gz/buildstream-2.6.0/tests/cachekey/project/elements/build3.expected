492953d9bdef66486c5111caf48f015690848e46d7f8de0f71bde386b68d4e5e